src/templates/gwfcore/gwfcore-launch-template.template.yaml

```diff
@@ -94,15 +94,72 @@ Resources:
                 --==BOUNDARY==
                 Content-Type: text/cloud-config; charset="us-ascii"
 
+                #cloud-config
+                repo_update: true
+                repo_upgrade: all
+
                 packages:
                 - jq
                 - btrfs-progs
                 - sed
                 - git
                 - amazon-ssm-agent
                 - unzip
+                - amazon-cloudwatch-agent
 
                 runcmd:
+
+                # create the amazon-cloudwatch-agent config file
+                - |
```

Comment:
Suggest using the `write_files` directive instead to create the CW config:
https://cloudinit.readthedocs.io/en/latest/topics/examples.html#writing-out-arbitrary-files